src/moveit_visual_tools.cpp

```diff
@@ -511,10 +521,8 @@ bool MoveItVisualTools::publishAnimatedGrasp(const moveit_msgs::Grasp& grasp,
     // Convert eigen pre-grasp position back to regular message
     pre_grasp_pose = tf2::toMsg(pre_grasp_pose_eigen);
 
-    // publishArrow(pre_grasp_pose, moveit_visual_tools::BLUE);
     publishEEMarkers(pre_grasp_pose, ee_jmg);
-    if (batch_publishing_enabled_)
-      trigger();
+
```

Comment:
it took me  a while to get the difference between `publishMarker` which would need a trigger in this case and `publishMarkers` which does not follow the `batch_publishing_enabled_` and therefore does not need a separate trigger/publish ever

Reply:
yea, its counter intuitive for sure. i'll make a TODO, but i can't address this now...

Reply:
Actually i just went ahead and fixed it: https://github.com/PickNikRobotics/rviz_visual_tools/pull/123